Stubs don't work with numpy 2

mypy fails:

```
/tmp/tmpukd3yzxb/.venv-393476311071167225/lib/python3.11/site-packages/numpy/ma/extras.pyi:2: error: Cannot find implementation or library stub for module named "numpy.lib.index_tricks"  [import-not-found]
/tmp/tmpukd3yzxb/.venv-393476311071167225/lib/python3.11/site-packages/numpy/ma/extras.pyi:2: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

Affects:

* [ ] seaborn
* [ ] shapely
* [ ] tensorflow

--- a/stubs/seaborn/METADATA.toml
+++ b/stubs/seaborn/METADATA.toml
@@ -1,7 +1,6 @@
 version = "0.13.2"
 # Requires a version of numpy and matplotlib with a `py.typed` file
-# TODO: stubtest errors when using numpy 2
-requires = ["matplotlib>=3.8", "numpy>=1.20,<2", "pandas-stubs"]
+requires = ["matplotlib>=3.8", "numpy>=1.20", "pandas-stubs"]
 # matplotlib>=3.8 requires Python >=3.9
 requires_python = ">=3.9"
 upstream_repository = "https://github.com/mwaskom/seaborn"

--- a/stubs/shapely/METADATA.toml
+++ b/stubs/shapely/METADATA.toml
@@ -1,5 +1,4 @@
 version = "2.0.*"
 # Requires a version of numpy with a `py.typed` file
-# TODO: stubtest errors when using numpy 2
-requires = ["numpy>=1.20,<2"]
+requires = ["numpy>=1.20"]
 upstream_repository = "https://github.com/shapely/shapely"

--- a/stubs/tensorflow/METADATA.toml
+++ b/stubs/tensorflow/METADATA.toml
@@ -3,8 +3,7 @@
 version = "2.16.*"
 upstream_repository = "https://github.com/tensorflow/tensorflow"
 # requires a version of numpy with a `py.typed` file
-# TODO: stubtest errors when using numpy 2
-requires = ["numpy>=1.20,<2", "types-protobuf", "types-requests"]
+requires = ["numpy>=1.20", "types-protobuf", "types-requests"]
 extra_description = "Partially generated using [mypy-protobuf==3.6.0](https://github.com/nipunn1313/mypy-protobuf/tree/v3.6.0) and libprotoc 25.1 on tensorflow==2.16.1 ."
 partial_stub = true
 

